src/main/java/com/uber/cadence/worker/Worker.java

```diff
@@ -590,20 +593,20 @@ private String getStickyTaskListName() {
 
   public static class FactoryOptions {
     public static class Builder {
-      private boolean enableStickyExecution;
+      private boolean disableStickyExecution = false;
```

Comment:
No need to initialize to a default value.

Reply:
resolved